jobboard/views.py

```diff
@@ -1,5 +1,76 @@
 from django.shortcuts import render
+from .models import Job
+from datetime import date
+from student.models import Student
+from recruiter.models import Recruiter
+
+
+def get_content_if_user_is_student(request):
+    """
+    If the user is a student then the jobs that will automatically
+    be displayed will be jobs that correspond to that student's degree
+    """
+    # use get methods once they get merged
+    user_as_student = Student.objects.get(user_id=request.user.id)
+    jobs_by_student_major = Job.get_jobs_by_major(user_as_student.major)
+    content = {'jobs_to_display': jobs_by_student_major,
+               'user': request.user,
+               'user_is_student': True,
+               'user_is_recruiter': False,
+               'user_as_student': user_as_student}
+    return content
+
+
+def get_content_if_user_is_recruiter(request):
```

Comment:
I made a "My Jobs" page for recruiters that displays all the jobs that the logged in recruiter has posted, as well as applicants for each job. As I recall this is what we agreed on in our last meeting.
I think it's better to redirect a recruiter to the "My Jobs" page when they log in instead of the view you created here.

Reply:
And I think we should still allow a recruiter to browse the jobs on the website, so when a recruiter goes to the main page they can be treated as if they're not logged in and see all the jobs from the past 6 months.

Reply:
> And I think we should still allow a recruiter to browse the jobs on the website, so when a recruiter goes to the main page they can be treated as if they're not logged in and see all the jobs from the past 6 months.

And yes that's a good idea. I will make these changes